ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,37 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
+ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+ RUN add-apt-repository "deb https://repo.sovrin.org/ursa/deb xenial master"
+
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
-    libindy-crypto=0.4.5 \
+
+    cmake \
+    autoconf \
+    libtool \
+    pkg-config \
+    libssl-dev \
     libindy=1.11.1~1343 \
 # rocksdb python wrapper
     libbz2-dev \
     zlib1g-dev \
     liblz4-dev \
     libsnappy-dev \
+    ursa \
     rocksdb=5.8.8
 
-RUN indy_ci_add_user $uid $user $venv
 
-RUN indy_image_clean
 
-USER $user
-WORKDIR /home/$user
+RUN curl -fsSL https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz | tar -xz
```

Comment:
Is there any others ways to install libsodium instead of downloading and compiling sources? Maybe we need to add some libsodium's repo or create a package and store it into our repo?

Reply:
Please note, that we already depend on libsodium. Do we have a version conflict here?

Reply:
```
└─ $ ▶ apt list --installed | grep libsodium
libsodium18/xenial,xenial-updates,xenial,now 1.0.8-5 amd64 [installed,automatic]
└─ $ ▶ apt show python3-nacl | grep -i depends
Depends: python3-cffi, python3 (<< 3.6), python3 (>= 3.5~), python3-cffi-backend-api-9729, python3-six, libc6 (>= 2.4), libsodium18 (>= 0.7.0)
```

Reply:
Ursa does not use libsodium at all. So any updates to libsodium would be tied to other features.